utils.js

```diff
@@ -20,6 +21,22 @@ const {
 //
 // These routines are for internal use, not for export to client.
 
+/**
+ * Return the named property, but only if it is an own property.
+ */
+function objectGetOwn(obj, prop) {
+  if (ObjectHasOwn(obj, prop))
+    return obj[prop];
+}
+
+/**
+ * Return the named options property, but only if it is an own property.
+ */
+function optionsGetOwn(options, longOption, prop) {
+  if (ObjectHasOwn(options, longOption))
+    return objectGetOwn(options[longOption], prop);
+}
```

Comment:
```suggestion
function optionsGetOwn(options, longOption, prop) {
  const optionConfig = objectGetOwn(options, longOption)
  if (optionConfig) return objectGetOwn(optionConfig, prop);
}
```

suggestion (non-blocking)

Reply:
I actually started with that, but switched to match my thinking at the time.